min_height: -0.43
max_height: 1.0
angle_min: -3.14
angle_max: 3.14
angle_increment: 0.0045 ##1440
scan_time: 0.1
range_min: 0.44
range_max: 20.0
